nature_news_scraper/run_scrape_gold_standard.sh

```diff
@@ -0,0 +1,10 @@
+#### this file is used to run the initial "gold standard" scrapes
+
+JSON_LINK_FILE="./links.json"
```

Comment:
How big is this links.json file? Also, is this file being used to crawl the news articles? If so, then I may recommend adding this to the repository instead of ignoring it to help with reproducibility (Can come in future PR. If there is a size issue feel free to ping me, dealt with this kind of headaches all the time).

Reply:
This is a good point. It is not big and since it'll be used for the benchmark I think this is a good idea.